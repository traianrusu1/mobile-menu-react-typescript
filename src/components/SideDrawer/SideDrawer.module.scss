@import "../../appStyles/variables";

$drawerWidth: 200px;

.sideDrawer {
  position: fixed;
  left: 0;
  right: 0;
  width: $drawerWidth;
  min-height: 600px;
  background-color: $appColor1;
}
