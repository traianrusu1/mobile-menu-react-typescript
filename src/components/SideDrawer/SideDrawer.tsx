import React from 'react';
import styles from './SideDrawer.module.scss';

interface Props {
  myProp: string;
}

const SideDrawer: React.FC<Props> = ({ myProp }: Props) => {
  return (
    <div className={ styles.sideDrawer }>
      SideDrawer
      {myProp}
    </div>
  );
};

export default SideDrawer;
