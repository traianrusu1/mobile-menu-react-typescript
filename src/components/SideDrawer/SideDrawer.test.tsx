import React from 'react';
import { render, cleanup } from '@testing-library/react';

import SideDrawer from './SideDrawer';

afterEach(cleanup);

describe('<SideDrawer />', () => {
  test('renders', () => {
    const { container } = render(<SideDrawer myProp="" />);
    expect(container).toMatchSnapshot();
  });
});

