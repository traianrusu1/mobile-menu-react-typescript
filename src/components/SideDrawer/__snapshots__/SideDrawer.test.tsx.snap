// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SideDrawer /> renders 1`] = `
<div>
  <div
    class="sideDrawer"
  >
    SideDrawer
    
  </div>
</div>
`;
